app/home.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1 user-scalable=no">
-    <title>MetaMask</title>
+    <title>Brave Wallet</title>
```

Comment:
I think we can do this with webpack as part of the branding too. I think you can split this and the manifest.json out of this PR.